WordPress/wporg-main-2022
# Fix About page content and design issues

Fixes #188 

### Screenshots

| Monitor | Desktop | Tablet (landscape) | Mobile |
|--------|--------|-------|-------|
| ![localhost_8888_about-2_](https://user-images.githubusercontent.com/1017872/217419265-22215c4a-39c7-4694-850b-c291ad08dd9c.png) | ![localhost_8888_about-2_(Desktop)](https://user-images.githubusercontent.com/1017872/217419289-d0868fc3-cd3f-48fd-a60e-8497e0f87782.png) | ![localhost_8888_about-2_(iPad)](https://user-images.githubusercontent.com/1017872/217419322-87418223-4402-45b8-818b-25c13b3bdfa0.png) | ![localhost_8888_about-2_(Samsung Galaxy S20 Ultra)](https://user-images.githubusercontent.com/1017872/217419340-f40477ad-635f-4519-a560-e79e6fc95afd.png) |

### How to test the changes in this Pull Request:

1. Build the styles
2. Ensure you have a page at `about-2` (you may need to select the about page template)
3. View the page and check fluid resizing, alignment and copy as described in issue
